README.md

```diff
@@ -90,6 +90,11 @@ module.exports = function(config) {
       },
 
       verbose: true // output config used by istanbul for debugging
+    },
+    // To configure Istanbul API
+    instrumentationIstanbulReporter: {
```

Comment:
Rather than adding more top level config to the karma reporter, can you add this config under the `coverageIstanbulReporter` config. So 
```
coverageIstanbulReporter: {
  instrumentation: {
    'default-excludes': false
  }
}
```

Reply:
Done.